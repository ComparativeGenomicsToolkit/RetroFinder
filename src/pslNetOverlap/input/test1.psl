1640	69	0	0	0	0	0	0	-	AK122970.1-2	2160	13	1803	chr3	199501827	183065268	183067635	11	307,24,395,279,13,26,334,90,68,133,40,	357,665,691,1090,1369,1382,1408,1744,1834,1908,2107,	183065268,183065575,183065599,183065994,183066595,183066612,183066644,183066978,183067394,183067462,183067595,
574	269	0	0	0	0	0	0	+	AK123626.1-50	2811	17	1151	chr18	76117153	14795419	14798746	23	57,40,15,134,44,7,23,4,75,76,27,19,18,116,28,10,14,7,23,11,7,4,84,	17,76,117,132,271,320,327,350,354,429,505,532,551,569,718,817,892,955,962,992,1046,1061,1067,	14795419,14795476,14795516,14795535,14795669,14795713,14795728,14795752,14795757,14798282,14798361,14798390,14798412,14798441,14798557,14798585,14798595,14798609,14798617,14798640,14798651,14798658,14798662,
763	496	0	0	0	0	0	0	+	AB271694.1-46	2100	22	1679	chr3	199501827	11923019	11924687	33	25,16,92,12,9,78,34,86,75,11,312,6,64,11,11,20,19,38,14,42,12,3,9,25,23,13,69,22,21,17,22,31,17,	22,47,70,168,180,196,274,339,425,502,513,830,844,1147,1159,1170,1191,1248,1286,1300,1346,1363,1366,1377,1402,1425,1439,1549,1571,1592,1609,1631,1662,	11923019,11923045,11923061,11923153,11923171,11923180,11923262,11923326,11923416,11923491,11923504,11923816,11923822,11924047,11924058,11924070,11924090,11924109,11924150,11924179,11924221,11924233,11924239,11924248,11924275,11924316,11924329,11924534,11924558,11924591,11924610,11924634,11924670,
513	225	0	0	0	0	0	0	-	AK126539.1-23	3552	2409	3517	chr16	88827254	29583882	29584638	36	54,134,73,9,13,15,12,19,17,16,27,21,10,25,15,23,10,27,11,6,4,22,14,16,8,7,11,4,8,8,13,21,9,6,14,36,	35,90,225,355,377,407,428,455,507,540,591,626,647,657,691,708,745,775,809,821,849,869,895,924,950,964,983,994,999,1010,1020,1033,1054,1066,1079,1107,	29583882,29583936,29584070,29584143,29584152,29584165,29584180,29584192,29584211,29584228,29584244,29584271,29584293,29584307,29584332,29584347,29584370,29584380,29584407,29584418,29584424,29584428,29584450,29584464,29584480,29584488,29584495,29584515,29584519,29584527,29584535,29584550,29584573,29584582,29584588,29584602,
529	317	0	0	0	0	0	0	-	AB354586.1-13	1195	246	1189	chr22	49691432	17364726	17365603	18	38,283,54,10,129,72,9,15,9,7,8,12,64,20,23,10,71,12,	6,74,358,416,437,566,638,651,666,679,703,729,741,807,828,851,866,937,	17364726,17364764,17365047,17365101,17365111,17365243,17365322,17365331,17365349,17365358,17365365,17365373,17365389,17365453,17365473,17365503,17365513,17365591,
569	322	0	0	0	0	0	0	+	AB354586.1-15	1195	208	1195	chr22	49691432	19373790	19374709	28	15,13,7,14,66,10,30,21,28,16,9,15,15,3,6,9,22,41,7,27,31,88,28,43,238,16,31,42,	208,223,239,246,260,327,337,368,390,421,438,461,482,503,510,520,529,551,595,602,629,661,766,794,838,1086,1104,1153,	19373790,19373809,19373822,19373831,19373852,19373918,19373931,19373961,19373982,19374010,19374026,19374035,19374050,19374065,19374068,19374074,19374086,19374111,19374152,19374160,19374190,19374221,19374309,19374339,19374382,19374620,19374636,19374667,
564	327	0	0	0	0	0	0	+	AB354586.1-17	1195	208	1195	chr22	49691432	19966641	19967560	26	15,13,7,14,66,10,30,21,28,16,9,15,15,3,6,9,22,41,7,27,23,96,28,43,282,45,	208,223,239,246,260,327,337,368,390,421,438,461,482,503,510,520,529,551,595,602,629,653,766,794,838,1150,	19966641,19966660,19966673,19966682,19966703,19966769,19966782,19966812,19966833,19966861,19966877,19966886,19966901,19966916,19966919,19966925,19966937,19966962,19967003,19967011,19967041,19967064,19967160,19967190,19967233,19967515,
533	325	0	0	0	0	0	0	+	AB354586.1-23	1195	239	1195	chr5	180857866	68963044	68963930	22	7,27,53,10,30,30,16,16,9,15,15,6,9,21,42,7,27,129,4,58,279,48,	239,246,273,327,337,368,405,421,438,461,495,510,520,529,550,595,602,629,770,779,838,1147,	68963044,68963053,68963087,68963140,68963153,68963183,68963213,68963232,68963248,68963257,68963272,68963290,68963296,68963308,68963332,68963374,68963382,68963412,68963541,68963545,68963603,68963882,
535	323	0	0	0	0	0	0	+	AB354586.1-24	1195	239	1195	chr5	180857866	68984332	68985218	22	7,27,53,10,30,30,16,16,9,15,15,6,9,21,42,7,27,129,4,58,279,48,	239,246,273,327,337,368,405,421,438,461,495,510,520,529,550,595,602,629,770,779,838,1147,	68984332,68984341,68984375,68984428,68984441,68984471,68984501,68984520,68984536,68984545,68984560,68984578,68984584,68984596,68984620,68984662,68984670,68984700,68984829,68984833,68984891,68985170,
535	323	0	0	0	0	0	0	-	AB354586.1-25	1195	239	1195	chr5	180857866	69234400	69235286	22	48,279,58,4,129,27,7,42,21,9,6,15,15,9,16,16,30,30,10,53,27,7,	0,78,358,421,437,566,593,603,645,666,679,685,719,748,758,774,797,828,858,869,922,949,	69234400,69234448,69234727,69234785,69234789,69234921,69234949,69234956,69235001,69235025,69235034,69235043,69235058,69235073,69235082,69235101,69235117,69235147,69235180,69235190,69235250,69235279,
