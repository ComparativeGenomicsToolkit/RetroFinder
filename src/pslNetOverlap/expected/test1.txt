#qName	/hive/users/baertsch/retro/hg18/exp/age/netCanFam2.txt	/hive/users/baertsch/retro/hg18/exp/age/netMm9.txt	/hive/users/baertsch/retro/hg18/exp/age/netRheMac2.txt	/hive/users/baertsch/retro/hg18/exp/age/netPanTro2.txt	
AK122970.1-2	95	43	100	0	
AK123626.1-50	9	70	68	120	
AB271694.1-46	6	6	0	0	
AK126539.1-23	0	0	0	0	
AB354586.1-13	91	100	0	0	
AB354586.1-15	97	49	0	0	
AB354586.1-17	29	15	0	0	
AB354586.1-23	96	99	100	120	
AB354586.1-24	96	99	45	120	
AB354586.1-25	96	35	100	120	
